Joplin_install_and_update.sh

```diff
@@ -117,7 +117,7 @@ fi
 #-----------------------------------------------------
 print "Checking dependencies..."
 ## Check if libfuse2 is present.
-LIBFUSE=$(ldconfig -p | grep "libfuse.so.2" || echo '')
+LIBFUSE=$(find /lib /usr/lib /lib64 /usr/lib64/ -name "libfuse.so.2"  2>/dev/null | grep "libfuse.so.2" || echo '')
```

Comment:
```suggestion
if [[ `command -v ldconfig` ]]; then
	LIBFUSE=$(ldconfig -p | grep "libfuse.so.2" || echo '')
else
	LIBFUSE=$(find /lib /usr/lib /lib64 /usr/lib64/ -name "libfuse.so.2" 2>/dev/null | grep "libfuse.so.2" || echo '')
fi
```

Reply:
Nitpick: Maybe we could use `if [[ $(command -v ldconfig) ]]; then` so we use don't mix different syntax variants in a single if/else block

Reply:
sure